releasenotes/notes/upgrade-run-circuits-backend-name-0dfce7eb558f224f.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    :meth:`qiskit_ibm.IBMProvider.run_circuits` method now takes a `backend_name`
+    instead of a `Backend` object.
```

Comment:
```suggestion
    :meth:`qiskit_ibm.IBMProvider.run_circuits` method now takes a `backend_name`
    parameter, which is a string, instead of `backend`, which is a ``Backend`` object.
```